### ISSUE TITLE
The Last of Us VRAM usage increases with dxvk-nvapi enabled

### ISSUE THREAD

**Issue by shelterx** [jp7677/dxvk-nvapi#210]
For some reason the VRAM usage increases massively in TLOU when NVAPI is enabled.
Tested with Nvidia RTX 4070

Not sure if there's anything that can be done about it.

**Comment by Blisto91** [comment]
Which Proton version was it tested with

**Comment by jp7677** [comment]
> For some reason the VRAM usage increases massively in TLOU when NVAPI is enabled. Tested with Nvidia RTX 4070.

Could you please specify this? Does this increased usage create troubles?

Additionally to Blisto’s question about your setup, having a log from the nvapi usage would be nice, can you try to create a trace log? See https://github.com/jp7677/dxvk-nvapi?tab=readme-ov-file#tweaks-debugging-and-troubleshooting

**Comment by shelterx** [comment]
Not really any issues, apart from hogging VRAM. 

It's been like this for a while, so it's not new but I thought I'd shine light on it.
I'm currently running Proton 9.
[dxvk-nvapi.log](https://github.com/user-attachments/files/16824265/dxvk-nvapi.log)

With dxvk-nvapi:
![image](https://github.com/user-attachments/assets/801591fa-f7e3-4f3d-a394-3fbffda97eea)

Without dxvk-nvapi:
![image](https://github.com/user-attachments/assets/30d698b9-a3a3-4087-8012-3c29bac5189f)

The screenshots are from a save game location I loaded but the the increased memory usage is actually showing right at the main menu too.

**Comment by Blisto91** [comment]
Could you give it a try with Experimental Bleeding Edge. A function related to reporting VRAM have recently been implemented.

**Comment by jp7677** [comment]
Thanks, I’m seeing that you are using the latest dxvk-nvapi from git and the game also uses the memory reporting function Blisto is referring to.
Seeing the increase from 6.2 to 8.6 and the note from Blisto that the grass is greener (I mean higher quality) and this doesn’t creates issues, I would also say that this works as intended. Thus seems like the increased VRAM usage is actually a result of fixing the memory reporting in recent nvapi and the game thus being able to run at better quality settings.

**Comment by shelterx** [comment]
Yes, thank you @Blisto91 for having good eyes.
I set the graphics settings to Ultra for both screenshots, however ... with dxvk-nvapi disabled, some texture settings are locked and it doesn't show what it defaults to.
The game got shitloads of graphics settings, so I did not notice that. I assumed Ultra would yield the same settings, apparently not.

So I'd say It's working as intended. Sorry for the buzz.
